src/App.js

```diff
@@ -91,8 +102,10 @@ export default function Game() {
 
   let moves = history.map((squares, move) => {
```

Comment:
You don't use this argument. Can you delete it?

Reply:
I can't remove it because of the default order of arguments in the callback function for the map method (element, index, array) and I need index/move

Reply:
You can use just an element, and it will be fine.